packages/presets/near-operation-file/src/utils.ts

```diff
@@ -70,3 +70,29 @@ export function resolveRelativeImport(from: string, to: string): string {
   }
   return fixLocalFile(clearExtension(relative(dirname(from), to)));
 }
+
+
+export function resolveImportPathFromPackage(baseDir: string, absPath: string | string[], relToPath: string | string[] = []): string {
+  const leftParts: string[] = Array.isArray(absPath) ? absPath : absPath.split('/');
```

Comment:
I thought windows users used unix notation in the context of nodejs projects?.... No worries I'll fix.. have a few other tweaks since the PR anyways...

Reply:
Not sure actually. I know @ardatan had issues with those in `graphql-toolkit`, so I guess paths still should use `sep`.

Reply:
Pretty sure its`/` everywhere, else every node module would be broken trying to do an import.   This is for creating source files, not for accessing files, in which case using the OS separator would absolutely be correct.  Only mentioning this because it could cause bugs --- and OS dependent code.

Reply:
@jspears On Windows, absolute paths are returned as escaped '\\' character even if import paths are used in the same way with unix. Using path.sep would be better and easier and also you can benefit from path.join, path.resolve and unixify packages instead of concatenating strings.